src/main/java/org/vanilladb/bench/StatisticMgr.java

```diff
@@ -237,7 +248,7 @@ private void outputTimelineReport(String fileName) throws IOException {
 		}
 	}
 
-	private void addTxnLatency(TxnResultSet rs) {
+	private void analyzeReadWriteSet(TxnResultSet rs) {
```

Comment:
Why did this name become `analyzeReadWriteSet`?

Reply:
It could be slightly ambiguous using addTxnLatency because it does more than adding txn latency, for example, adding commited tx count.

analyzeReadWriteSet is ambigous though.